jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -719,3 +720,24 @@ function configure_server_state() {
 function generate_random_id() {
     cat /dev/urandom | env LC_CTYPE=C tr -dc 'a-zA-Z0-9' | fold -w 16 | head -n 1
 }
+
+function configure_jbpm_cluster(){
+    if [ "${KIE_SERVER_JBPM_CLUSTER^^}" = "TRUE" ]; then
+      configure_jbpm_cache
+      log_info "KIE_SERVER_JBPM_CLUSTER enabled"
+    else
+      log_info "KIE_SERVER_JBPM_CLUSTER disabled"
+    fi
+}
+
+function configure_jbpm_cache() {
+    sed -i 's#<cache-container name="server" aliases="singleton cluster" default-cache="default" module="org.wildfly.clustering.server">#<cache-container name="jbpm">\
+        <transport lock-timeout="60000"/>\
```

Comment:
Some of the values may be good to be editable, like this timeout. For some users this limit may not be enough, and they would prefer higher value set there. For now I think it would be ok to just have Kie Server's env variable for configuration like this (not including it, for now, to Operator)

Reply:
maybe the name and lock timeout

Reply:
IS the configuration used in the on-premise deployment https://issues.redhat.com/browse/JBPM-9399

Reply:
Yes, but it doesn't mean that we can't make it configurable on the container image.

Reply:
Done